core/src/bin/run.ts

```diff
@@ -93,6 +99,44 @@ export class RunCLI extends CLI {
     }
   }
 
+  async checkSchedules(scheduleIds?: boolean | string[]) {
+    if (typeof scheduleIds === "undefined") return;
+    if (typeof scheduleIds === "boolean") {
+      return GrouparooCLI.logger.fatal(
+        `Please specify which schedule ids to run`
+      );
+    }
+
+    const schedules = await Schedule.findAll({ where: { id: scheduleIds } });
+    const foundScheduleIds = schedules.map((s) => s.id);
+    scheduleIds.forEach((id) => {
+      if (!foundScheduleIds.includes(id))
+        return GrouparooCLI.logger.fatal(
+          `Schedule with id "${id}" was not found`
+        );
+    });
+  }
+
+  async stopScheduleRuns() {
```

Comment:
I wonder if we should stop other types of runs too: Group, Property, Internal Run...

Reply:
From standup discussion: we'll only worry about Schedules for now